Tosca Tricentis Documentation (#445)
saucelabs/sauce-docs: docs/basics/integrations/tosca.md

---
id: tosca
title: "Tosca Tricentis™ Integration"
sidebar_label: "Tosca Tricentis"
keywords:
- tosca
- tricentis
- quickstart
- how-to
---

<p><small><span class="highlight beta">BETA</span></small></p>

[Tosca's Tricentis™](https://www.tricentis.com/) is one of the world's leading enterprising enterprise software testing platforms. Tricentis is "totally automated, fully codeless, and intelligently driven by AI". The Sauce Labs and Tosca Tricentis integrations allows you to run your Tricentis tests on the Sauce Labs platform—giving you access to all of the [features that the Sauce Labs continuous testing cloud](https://saucelabs.com/platform) has to offer.

Below is a guide to set up the Sauce Labs integration. This integration allows you to run your Tricentis tests on our platform.

## What You'll Need

* [A Sauce Labs Account](https://saucelabs.com/sign-up)
* [A Tosca Tricentis™ Account](https://www.tricentis.com/software-testing-tool-trial-demo/)

## Configure the Commander View

* In the Tricentis™ UI, navigate to **Event View** > **Commander View**.

> * Set the `Url` to Sauce Lab's data center endpoint: `https://ondemand.sauclabs.com`

Author: ```suggestion
* Set the `Url` to Sauce Lab's data center endpoint: `https://ondemand.saucelabs.com`
```

  :::note Data Center Location

>   Selecting the appropriate data center in which to run your tests is important. Please read the following [documentation](https://wiki.saucelabs.com/display/DOCS/Data+Center+Endpoints) to learn more about setting your tests configuration options to the correct data center location.

Author: ```suggestion
  Selecting the appropriate data center in which to run your tests is important. Please read the following [documentation](https://docs.saucelabs.com/basics/data-center-endpoints/data-center-endpoints/index.html#endpoints) to learn more about setting your tests configuration options to the correct data center location.
```

  :::

> If you're using a proxy for your tests you must also point the upstream address to the Sauce Labs data center endpoint.
> 

Reviewer: ```suggestion
If you are using a proxy for your tests you must also point the upstream address to the Sauce Labs data center endpoint.

```
